apps/server/src/modules/files-storage/uc/files-storage.uc.spec.ts

```diff
@@ -651,20 +651,17 @@ describe('FilesStorageUC', () => {
 				);
 			});
 
-			it('should return empty response if entities not found', async () => {
+			it('should return exception response if entities not found', async () => {
 				fileRecordRepo.findBySchoolIdAndParentId.mockResolvedValue([[], 0]);
 
-				const res = await service.copyFilesOfParent(userId, sourceParentParams, copyFilesParams);
-				expect(res).toEqual([[], 0]);
+				await expect(service.copyFilesOfParent(userId, sourceParentParams, copyFilesParams)).rejects.toThrow();
```

Comment:
No it should not throw an error. The copy service alway try to want copy any item in is tree. Many of them has no representation in file service and should not run into an error. 
Or test is any other case.